dracula-theme.jsonc

```
@@ -6,35 +6,49 @@
   "themeValues": {
     // values applied to the entire app
     "root": {
+      "current__line": "#44475a",
+      "background": "#282a36",
+      "foreground": "#f8f8f2",
+      "comment": "#6272a4",
+      "pink": "#ff79c6",
       "red": "#ff5555",
       "orange": "#ffb86c",
       "yellow": "#f1fa8c",
       "green": "#50fa7b",
-      "purple": "#bd93f9",
-      "pink": "#ff79c6",
-      "cyan": "#8be9fd",
-      "app__bg0": "#282a36",
-      "comment": "#6272a4",
+      "blue": "#bd93f9",
+      "ltblue": "#8be9fd",
```

Same with this. We should keep the same naming convention that Dracula does in [this list](https://draculatheme.com/contribute). This should stay cyan
```suggestion
      "cyan": "#8be9fd",
```